assets/javascripts/kitten/components/typography/title/index.js

```diff
@@ -1,31 +1,27 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import styled, { css } from 'styled-components'
+import styled from 'styled-components'
+import classNames from 'classnames'
+import COLORS from '../../../constants/colors-config'
 import TYPOGRAPHY from '../../../constants/typography-config'
 import { modifierStyles } from './helpers/modifier-styles'
 
 const StyledTitle = styled.span`
+  --Title-css-color: ${COLORS.font1};
+
   ${TYPOGRAPHY.fontStyles.bold};
-  ${({ modifier }) => modifierStyles(modifier)}
+  color: var(--Title-css-color);
```

Comment:
Comme c'est un composant particulièrement important je rajouterais ça pour que les banquiers de Lendopolis ne soient pas trop déboussolés sur leur IE11. (avec le commentaire pour le supprimer quand on supprimera tous les fix IE11)

```suggestion
    color: ${COLORS.font1}; /* IE11 */
    color: var(--Title-css-color);
```